packages/gatsby/src/cache-dir/app.js

```diff
@@ -8,6 +8,13 @@ window.___emitter = require(`./emitter`)
 // Let the site/plugins run code very early.
 apiRunner(`onClientEntry`)
 
+if (process.env.NODE_ENV === `development`) {
```

Comment:
Ah, good to know. I wasn't sure, but I guess I should have taken a closer look at the webpack config 😁